src/types.ts

```diff
@@ -19,7 +19,9 @@ import {
   ArweaveSigner,
   DataItemCreateOptions,
 } from 'arbundles';
+import * as Transaction from 'arweave/node/lib/transaction.js';
```

Comment:
watch out for named export here - it may not be compatible with web runtimes and bc it's in a shared export file (not in node or web) it may cause runtime issues

Reply:
not sure how to solve this one atm. this export is hard to use. maybe declare the whole type on our end?

Reply:
this one should be okay for now